src/DependencyInjection/SymfonyTraceExtension.php

```diff
@@ -38,8 +38,14 @@
  *          response_header: string,
  *          generator_service: ?string,
  *      },
- *      trust_request_header: bool,
- *      send_response_header: bool,
+ *      request: array{
+ *          trust_header: bool,
+ *          trusted_ips: string[]|string,
+ *      },
+ *      response: array{
+ *          send_header: bool,
+ *          trusted_ips: string[]|string,
```

Comment:
I think this should be `string[]|string|null`, because of `->defaultNull()` in the configuration setup